nvtabular/utils.py

```diff
@@ -121,3 +123,36 @@ def report(chunk, chunksize, total):
         with gzip.open(local_filename, "rb") as input_file:
             with open(local_filename[:-3], "wb") as output_file:
                 shutil.copyfileobj(input_file, output_file)
+
+
+def _ensure_optimize_dataframe_graph(ddf=None, dsk=None, keys=None):
```

Comment:
I have a (potentially irrational) vendetta against the word "ensure", but as much as I want to, I really can't argue with this usage. Totally accurate. :+1:

Reply:
:) Happy to rename it to `_optimize_dataframe_graph` if you'd like

Reply:
I considered suggesting that, but on reflection, I concluded it would be less accurate/informative. (Mostly left this comment to signal that I'd actually read and tried to think about the code, because a LGTM with no comments is kind of sus.)